packages/plugin-ext/src/hosted/browser/worker/worker-main.ts

```diff
@@ -55,32 +53,49 @@ rpc.set(MAIN_RPC_CONTEXT.HOSTED_PLUGIN_MANAGER_EXT, new HostedPluginManagerExtIm
                 console.error(`WebWorker: Cannot start plugin "${plugin.model.name}". Frontend plugin not found: "${plugin.lifecycle.frontendModuleName}"`);
                 return;
             }
-            startPlugin(plugin, ctx[plugin.lifecycle.frontendModuleName], plugins);
+            return ctx[plugin.lifecycle.frontendModuleName];
         }
     },
-    stopPlugins(contextPath: string, pluginIds: string[]): void {
-        pluginIds.forEach(pluginId => {
-            const pluginData = plugins.get(pluginId);
-            if (pluginData) {
-                // call stop method
-                if (pluginData.stopPluginMethod) {
-                    pluginData.stopPluginMethod();
+    init(raw: PluginMetadata[]): [Plugin[], Plugin[]] {
```

Comment:
should we use something more meaningful than raw if we know it's of type pluginMetadata ?